test/do.js

```diff
@@ -3,6 +3,8 @@
 const metasync = require('..');
 const metatests = require('metatests');
 
+// Emulate Asynchronous calls of function
+//   callback - <Function>
 const wrapAsync = callback => {
   setTimeout(callback, Math.floor(Math.random() * 500));
 };
```

Comment:
test.error on line 48 of this file?

Reply:
Will make this after land https://github.com/metarhia/metasync/pull/390